scripts/detect_duplicate_features.py

```diff
@@ -67,15 +48,18 @@ def find_overlapping_rules(new_rule_path, rules_path):
 
     # Loads features of new rule in a list.
     new_rule_features = get_features(new_rule_path)
-
     count = 0
     overlapping_rules = []
 
     # capa.rules.RuleSet stores all rules in given paths
     ruleset = capa.main.get_rules(rules_path)
 
     for rule_name, rule in ruleset.rules.items():
-        rule_features = get_child_features(rule.statement)
+        rule_features = set()
+        if isinstance(rule.statement, ceng.Statement):
+            rule_features = rule.statement.get_all_features()
+        else:
+            rule_features.add(rule.statement)
```

Comment:
would you move `get_all_features` onto the `Rule` class so it can encapsulate this repeated logic?